libs/designsystem/modal/src/modal/services/modal-animation-builder.service.ts

```diff
@@ -17,7 +16,7 @@ export class ModalAnimationBuilderService {
   };
 
   public enterAnimation(currentBackdrop?: HTMLIonBackdropElement): AnimationBuilder {
-    return (baseEl: HTMLElement, opts: ModalAnimationOptions): Animation => {
+    return (baseEl: HTMLElement, opts: any): Animation => {
```

Comment:
This looks funky, but between v6.7.5 and v7.3.3, the core package exports was changed from default exports to named exports, and now `ModalAnimationOptions` is only available internally to Ionic. 

v6.7.5:
`export * from './components/modal/modal-interface';`

v7.3.3:
`export { ModalOptions, ModalCustomEvent } from './components/modal/modal-interface';`

Reply:
So, should we copy out the type from Ionic core, or continue with `any`?

Reply:
@RasmusKjeldgaard maybe just inline a type with the property from `ModalAnimationOptions` that we need?
```suggestion
    return (baseEl: HTMLElement, opts: { presentingEl?: HTMLElement }): Animation => {
```

Reply:
Excellent, good idea!